docs/configuration/monitoring.md

```diff
@@ -58,6 +58,28 @@ Crow is a visualization tool that displays information about the projects and jo
 
 The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
 
+By default, the `crow` tool can be accessed through the `/monitoring` path of the url pointing to your Codacy installation, e.g. `http://codacy.company.org/monitoring`.
+It is possible for you to configure this path through the `values.yaml` file in the `crow` and `codacy-ingress` sections. You must also let `crow` know which is your codacy's url, so that it can properly anchor links to your projects.
+
+```yaml
+  crow:
+  config:
```

Comment:
I would remove this and just add the news paths on the table in the main README.md. Only keep here what is essential